OIPA/iati/models.py

```diff
@@ -283,15 +296,6 @@ def __unicode__(self,):
         return "%s - %s" % (self.code, self.name)
 
 
-class TransactionType(models.Model):
```

Comment:
I'm pretty sure.
Check this line:
https://github.com/openaid-IATI/OIPA/pull/123/files#diff-ab1e0e551bf27c10e044e8cf151cfd90R5